src/autorest-core/lib/ref/yaml.ts

```diff
@@ -90,6 +90,9 @@ export function ResolveAnchorRef(yamlAstRoot: YAMLNode, anchorRef: string): YAML
  * Populates yamlNode.valueFunc with a function that creates a *mutable* object (i.e. no caching of the reference or such)
  */
 function ParseNodeInternal(yamlRootNode: YAMLNode, yamlNode: YAMLNode, onError: (message: string, index: number) => void): () => any {
+  if (!yamlNode) {
```

Comment:
cosmetic fix: js-yaml seems to sometimes use `null` to represent `null` in YAML, although there is a corresponding `YAMLNode` instantiation for that too... Occured for me for the first array item of:
```
asd:
- 
- null
```
Note that both array items are valid and identical.